jenkins-jobs/job-dsl/release_debezium_stage_prod.groovy

```diff
@@ -0,0 +1,41 @@
+pipelineJob('release-debezium-stage-prod') {
+    displayName('Debezium Stage Product Artifacts')
+    description('Uploads product artifacts into stage location')
+
+    properties {
+        githubProjectUrl('https://github.com/debezium/debezium')
+    }
+
+    logRotator {
+        numToKeep(5)
+    }
+
+    parameters {
+        stringParam('BUILD_VERSION', '', 'Maven artifact id of the product binaries')
+        stringParam('BUILD_VERSION_INCUBATOR', '', 'Maven artifact id of the product binaries in incubator')
+        stringParam('PRODUCT_VERSION', '', 'Product version')
+        stringParam('USERNAME', '', 'Username to log to staging host')
+        stringParam('PASSWORD', 'master', 'Password to log to staging host')
+        stringParam('CONNECTORS', 'mysql postgres mongodb sqlserver', 'The list of released connectors')
+        stringParam('CONNECTORS_INCUBATOR', 'db2', 'The list of released incubating connectors')
+        stringParam('STAGE_FILES', 'false', 'When checked the uploaded artifacts are staged')
+        stringParam('SOURCE_MAVEN_REPO', 'debezium-prod-repo', 'Maven repository URL with product artifacts')
+            credentialsParam('TARGET_HOST') {
+            type('org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl')
+            required()
+            defaultValue('debezium-staging-url')
+            description('Staging host')
+        }
+        stringParam('ARTIFACT_DIR', '/mnt/rcm-guest/staging/amq', 'Staging directory')
+    }
+
+    definition {
+        cps {
+            script(readFileFromWorkspace('jenkins-jobs/pipelines/prod-release-pipeline.groovy'))
+        }
+    }
+
+    publishers {
```

Comment:
this needs to be moved into the pipeline as pipeline job in job DSL does not have publisher